contrib/completion/zsh/_docker

```diff
@@ -2713,6 +2713,8 @@ __docker_subcommand() {
                         __docker_complete_nodes && ret=0
                     elif [[ ${words[(r)--type=plugin]} == --type=plugin ]]; then
                         __docker_complete_plugins && ret=0
+                    elif [[ ${words[(r)--type=service]} == --type=secrets ]]; then
```

Comment:
Oh, thanks of the catch; can you open a pull request to fix that, or want someone else to do that?